### ISSUE TITLE
Angular

### ISSUE THREAD

**Issue by fifiuro** [davicotico/Menu-Editor#2]
Consulta se puede implementar el Menú Editor en Angular

**Comment by davicotico** [comment]
Sí, funciona correctamente. Puedes seguir los siguientes pasos:
1. Primero instalas en tu proyecto la biblioteca desde NPM
```
npm install @davicotico/menu-editor
```
2. Se debe incluir un elemento `div` en el template html del componente en el cual se mostrará el editor de menú.
```html
<div id="myEditor"></div>
```
3. Importar la clase `MenuEditor` en el archivo typescript del componente

```ts
import { MenuEditor } from '@davicotico/menu-editor';
```
4. Al inicializar el componente (en el evento/hook OnInit ) tiene que crearse una instancia  de la clase `MenuEditor` y a partir de ahí setear los datos y eventos que correspondan para finalmente ejecutar el método `mount()`.
```ts
export class AppComponent implements OnInit {

  menu!: MenuEditor;

  constructor() {

  }

  ngOnInit(): void {
    this.menu = new MenuEditor('myEditor');
    const datos =  [
      {
      "text": "Home",
      "href": "/home",
      "tooltip": "Go to home page",
      "icon": "fa-solid fa-house",
      "children": []
      },
      {
      "text": "About Us",
      "href": "/about",
      "tooltip": "Learn more about our company",
      "icon": "fa-solid fa-address-card",
      "children": []
      },
      {
      "text": "Services",
      "href": "/services",
      "tooltip": "Discover the services we offer",
      "icon": "fa-solid fa-gear",
      "children": [
        {
        "text": "Service 1",
        "href": "/services/1",
        "tooltip": "Details for Service 1",
        "icon": "cog",
        "children": []
        },
        {
        "text": "Service 2",
        "href": "/services/2",
        "tooltip": "Details for Service 2",
        "icon": "cog",
        "children": []
        }
      ]
      }
    ];
    this.menu.setArray(datos);
    this.menu.mount();
  }
}
```

5. Es importante incluir el archivo de estilos `styles.min.css`, sin embargo, acabo de ver que existe un bug en el paquete NPM y por eso (por ahora) tendrás que copiar el archivo desde el repositorio (está en `dist/styles.min.css`) y llevarlo a tu proyecto (puede ser a `src/assets`). En tu archivo `src/index.html` puedes incluirlo de esta forma:

```html
<link href="./assets/styles.min.css" rel="stylesheet">
```

**Comment by czele** [comment]
Hey Davi, I saw that there is a bug on the NPM package. When it changes. Please, let us know!

**Comment by davicotico** [comment]
Oi @czele 
Gostaria de pedir sua gentileza em descrever o bug que você encontrou na biblioteca.
Tenho um pull request por aprovar e vou atualizar o pacote no npm no máximo até o dia sábado. Posso resolver o bug também nesta release. Agradeço desde já pela sua atenção e ajuda.
(p.s. Pode me escrever em português mesmo, eu falo português)

**Comment by czele** [comment]
Oi @davicotico que legal que fala português.
Não é um bug especificamente. Eu segui o passo 5 que você descreveu para o Fifiuro, mas mesmo depois de importar o pacote de estilos no projeto, parece que ele não aplicou, pois o CSS não mudou em nada a vizualização do menu. 
Mas já que disse que vai arrumar esse bug fim de semana eu espero ele para testar o menu.
Muito obrigada pela resposta!

**Comment by davicotico** [comment]
@czele 
@fifiuro 
A new version of menu-editor is now available on npm. Also, you can find an example to get started [here](https://github.com/davicotico/menu-editor/tree/main/examples/bootstrap5)
Thank you for your support!
